src/main/javadoc/overview.html

```diff
@@ -17,6 +17,10 @@
 <head></head>
 <body>
 Java API for the <a href="http://nats.io">NATs messaging system</a>.
+<p>The Java API is hosted on <a href="https://github.com/nats-io/nats.java">Github NATs Java API</a>. Please check the <a href="https://github.com/nats-io/nats.java/blob/main/README.md"><b>README</b></a> for updates, hints and best practices.
+<p>Examples are included <a href="https://github.com/nats-io/nats.java/tree/main/src/examples/java/io/nats/examples">NATs Client API examples</a>.
+<p>More examples, including other programming languages, at <a href="https://natsbyexample.com/">NATs By Example</a>.
+
```

Comment:
Can you please add this also: https://github.com/nats-io/java-nats-examples. By github convention, if README.MD is found in the root of the repo, it is shown on the main page of the repo, so I don't know if you need to reference it's actual page.

Reply:
Will add to NATS by example.
I was pointing to README explicitely because that makes it harder to miss the instructions. When following the main project link the code is on the top.